source/agg-pipeline-builder/maxtime-ms-pipeline.txt

```diff
@@ -0,0 +1,65 @@
+.. _set-maxtime-ms-agg:
+
+=====================================
+Set maxTimeMS for Aggregation Queries
+=====================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Use the ``maxTimeMS`` parameter on the :guilabel:`Aggregations` tab to 
+specify an upper time limit in milliseconds for aggregation pipelines
+that run in |compass|.
+
+About This Task 
+---------------
+
+If you already specified a ``maxTimeMS`` value in the :ref:`Compass Settings 
+panel <compass-interface-settings>`, the :ref:`command line <cli-options>`, or 
+a :ref:`configuration file <config-file-options>`, the value you enter for 
+:guilabel:`Max Time MS` value must be lower than the limit you previously 
+specified.
+
+Steps 
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`More Options`.
+      
+      On the :guilabel:`Aggregations` tab, click :guilabel:`More Options`
+
+      .. figure:: /images/compass/agg-builder-click-more-options.png
+         :alt: More Options dropdown.
+
+   .. step:: Specify a Max Time MS value.
+
+      Next to the :guilabel:`Max Time MS` field, enter a numeric value to set as 
+      the maximum amount of time in milliseconds that an aggregation 
+      pipeline can run.  
+      
+      For a ``5`` second limit, enter ``5000``.
+
+      .. figure:: /images/compass/agg-builder-enter-maxtime-ms.png
+         :alt: Enter a maximum timeout value 
```

Comment:
I'm also not sure that this screenshot is necessary. I feel like the first screenshot where we point users to the location of the option is sufficient, but I'll leave it to you whether we want to keep this one as well.

Reply:
Initially I left it there since the first image doesn't show the dropped-down options menu with the parameter, but on second look I think you're right. Removing!